mmdet/datasets/openimages.py

```diff
@@ -563,6 +563,17 @@ def denormalize_gt_bboxes(self, annotations):
             annotations[i]['bboxes'][:, 1::2] *= h
         return annotations
 
+    def get_cat_ids(self, idx):
+        """Get category ids by index.
+
+        Args:
+            idx (int): Index of data.
+
+        Returns:
+            list[int]: All categories in the image of specified index.
+        """
+        return self.get_ann_info(idx)['labels'].astype(np.int).tolist()
```

Comment:
Seems `OpenImagesDataset` inherit `CustomDataset`. Is this part necessary?

Reply:
Yes, because of the keys are different, there are some differences between them: 
In OID, we use 
`self.get_ann_info(idx)['labels'].astype(np.int).tolist()` 
but in CustomDataset, we use 
`self.data_infos[idx]['ann']['labels'].astype(np.int).tolist()`